### PR TITLE
fix: Add tooltip popover for details on IDV attempts history

### PR DESCRIPTION
**Pull request by schenedx**
### Description
[MST-1383](https://openedx.atlassian.net/browse/MST-1383)

To help support team member to see more information about the IDV attempt associated with VerifiedName records, we are providing a pop over element to the VerifiedNameHistory Modal Dialog. See screenshot below. This is a minor enhancement.

tooltipforVerifiedNameIDV

### CODE REVIEW

File: src/users/v2/VerifiedName.jsx [hunk 1]
```diff
@@ -31,17 +34,48 @@ export default function VerifiedName({ username }) {
...
-  // Modal to display verified name history
-  const openVerifiedNameModal = (data) => {
-    const tableData = data.map(result => ({
+  const VerifiedNameHistoryTableData = useMemo(() => verifiedNameHistoryData.map(
+    result => ({
       verifiedName: result.verifiedName,
       profileName: result.profileName,
       status: result.status,
-      idvAttemptId: result.verificationAttemptId,
+      idvAttemptId: result.verificationAttemptId ? (
```
Comment (bseverino) [inline src/users/v2/VerifiedName.jsx:42]: Does this display for all statuses? If there's no message will there just be an empty popover?

Comment (bseverino) [reply src/users/v2/VerifiedName.jsx:42]: Oh, it will still have the status displayed, I answered my own question.

**Review by Ali-D-Akbar — APPROVED** [review]
Looks good to me. Popover is a nice design I must say.

**Review by bseverino — APPROVED** [review]
lgtm, I agree Popover works well here 👍

**Pull request merged**